src/test/java/ContinuousIntegrationServerTest.java

```diff
@@ -0,0 +1,27 @@
+import org.junit.jupiter.api.Test;
+
+import java.io.File;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ContinuousIntegrationServerTest {
+
+    @Test
+    void cloneTheProject() {
+        ContinuousIntegrationServer.cloneTheProject(ContinuousIntegrationServer.gitHubRepoHTTPS);
+        File file1 = new File("assessmentDir/src/main/java/HelloWorld.java");
+        assertTrue(file1.exists());
+        File file2 = new File("assessment/src/main/java/GoodbyeWorld.java");
+        assertFalse(file2.exists());
+    }
+
+
+    @Test
+    void build() {
```

Comment:
Can you make a test that assert `false`? I think the minimum for testing is having a true case and a false case to be sure that is doesn't always return the same value

Reply:
Yes, i'll fix that too!